statistical_clear_sky/__init__.py

```diff
@@ -1,2 +1,6 @@
 from statistical_clear_sky.algorithm.iterative_fitting import IterativeFitting
 from statistical_clear_sky.algorithm.iterative_fitting import IterativeFitting as SCSF
+
+from warnings import warn
+
+warn("Starting in Solar Data Tools 1.5.0, the Statistical Clear Sky package will be deprecated.", FutureWarning)
```

Comment:
@bmeyers I made up a version number here, is this fine or should we wait until 2.0? Also do we want to add any information about the replacement/reason here?

Reply:
I think want to say that it is depreciated _now_ (as of the next tagged release) and that it will be _removed_ in a future update (I think v2 makes sense)